docs/ide-usage.md

```diff
@@ -0,0 +1,100 @@
+
+   ```sh
+   ./gradlew setupEclipse
+   ```
+
+   **Verification:** The files `.project` and `.classpath` should be added to the project root directory.
+
+1. Start Eclipse and do the following works before importing the project:
+   * Google App Engine: Go to `Eclipse → Preferences → Google → App Engine`, click the `Add` button, and point it to where Gradle keeps the downloaded SDK.<br>
+     This directory can be found by running the command `./gradlew printUserHomeDir`.<br>
+     Further instructions for installing can be found [here](https://developers.google.com/eclipse/docs/using_sdks).
+
+     ![setupguide-2.png](images/setupguide-2.png)
+
+   * JRE: Go to `Eclipse → Preferences → Java → Installed JRE` and ensure a **JDK 1.7** (not JRE, not JDK 1.8) entry exists.
+
+     ![setupguide-3.png](images/setupguide-3.png)
+
+   Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
+
+1. Import the project to your Eclipse instance.
+   * Go to `File → Import...`.
+   * Select `Existing Projects into Workspace` under `General`.
+   * Set the `root directory` to the location where the repo is cloned.
+   * Click `Finish`.
+
+1. Configure the following project-specific settings (all can be found in `Project → Properties → ...`):
+   * Text encoding: `Resources` → change the `Text file encoding` setting from `Default` to `Other: UTF-8`.
+
+     ![setupguide-4.png](images/setupguide-4.png)
+
+   * Google App Engine: set up the following by going to `Google → ...`:
+     * Datanucleus version: `App Engine → Datastore → Datanucleus JDO/JPA Version` → select `v1`.
+
+       ![setupguide-5.png](images/setupguide-5.png)
+
+     * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
+     * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
+     * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
+     Configure for all the languages used in TEAMMATES:
+     * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
+     * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
+     * HTML: `Web → HTML Files → Editor → Indent using spaces`.
+     * CSS: `Web → CSS Files → Editor → Indent using spaces`.
+     * XML: `XML → XML Files → Editor → Indent using spaces`.
+   * Validation: We do not validate HTML, JSP, and XML.<br>
+     `Validation` → uncheck the `Build` option for `HTML Syntax Validator`, `JSP Content Validator`, `JSP Syntax Validator`, and `XML Validator`.
+
+1. `Clean` the project for all changes to take effect. Ensure that there is no errors. Warnings are generally fine to be ignored.
```

Comment:
`there is no` -> `there are no`
`Warnings are generally fine to be ignored` -> `Warnings are generally fine and can be ignored`